partiql-test-data/fail/parser/primitives/call.ion

```diff
@@ -0,0 +1,174 @@
+substring::[
+    sql92::[
+        parse::{
+            name: "SUBSTRING sql92 syntax missing left paren",
+            statement: "SELECT SUBSTRING FROM 'asdf' FOR 1) FROM foo",
+            assert: {
+                result: ParseError
+            },
+        },
+        parse::{
+            name: "SUBSTRING sql92 syntax missing FROM or comma",
+            statement: "SELECT SUBSTRING('str' 1) FROM foo",
+            assert: {
+                result: ParseError
+            },
+        },
+        parse::{
+            name: "SUBSTRING sql92 syntax without length and missing right paren",
+            statement: "SELECT SUBSTRING('str' FROM 1 FROM foo",
+            assert: {
+                result: ParseError
+            },
+        },
+        parse::{
+            name: "SUBSTRING sql92 syntax with length missing right paren",
+            statement: "SELECT SUBSTRING('str' FROM 1 FOR 1 FROM foo",
+            assert: {
+                result: ParseError
+            },
+        }
+    ],
+    partiql::[
+        parse::{
+            name: "SUBSTRING without length missing right paren",
+            statement: "SELECT SUBSTRING('str', 1 FROM foo",
+            assert: {
+                result: ParseError
+            },
+        },
+        parse::{
+            name: "SUBSTRING missing right paren",
+            statement: "SELECT SUBSTRING('str', 1, 1 FROM foo",
+            assert: {
+                result: ParseError
+            },
+        }
+    ]
+]
+
+trim::[
+    parse::{
+        name: "TRIM no left paren",
+        statement: "TRIM ' ')",
+        assert: {
+            result: ParseError
+        },
+    },
+    parse::{
+        name: "TRIM too many arguments",
+        statement: "TRIM(BOTH ' ' FROM 'test' 2)",
```

Comment:
This is implementation-dependent as to whether it's a 'parse' error or a type/semantic error